src/app/services/views/ServiceListView.vue

```diff
@@ -14,133 +14,149 @@
-                    {{ item.name }}
-                  </XAction>
-                </TextWithCopyButton>
-              </template>
-
-              <template #addressPort="{ row }">
-                <TextWithCopyButton
-                  v-if="row.addressPort"
-                  :text="row.addressPort"
-                />
-
-                <template v-else>
-                  {{ t('common.collection.none') }}
-                </template>
-              </template>
-
-              <template #online="{ row: item }">
-                <template
-                  v-if="item.dataplanes"
-                >
-                  {{ item.dataplanes.online || 0 }} / {{ item.dataplanes.total || 0 }}
-                </template>
-                <template
-                  v-else
-                >
-                  {{ t('common.collection.none') }}
-                </template>
-              </template>
-
-              <template #status="{ row: item }">
-                <StatusBadge :status="item.status" />
-              </template>
-
-              <template #details="{ row }">
-                <XAction
-                  class="details-link"
-                  data-testid="details-link"
-                  :to="{
-                    name: 'service-detail-view',
-                    params: {
-                      mesh: row.mesh,
-                      service: row.name,
-                    },
-                  }"
-                >
-                  {{ t('common.collection.details_link') }}
-
-                  <ArrowRightIcon
-                    decorative
-                    :size="KUI_ICON_SIZE_30"
-                  />
-                </XAction>
-              </template>
-            </AppCollection>
-          </KCard>
-
-          <RouterView
-            v-if="route.params.service"
-            v-slot="child"
-          >
-            <SummaryView
-              @close="route.replace({
-                name: 'service-list-view',
-                params: {
-                  mesh: route.params.mesh,
-                },
-                query: {
-                  page: route.params.page,
-                  size: route.params.size,
-                },
-              })"
-            >
-              <component
-                :is="child.Component"
-                :name="route.params.service"
-                :service="data?.items.find((item) => item.name === route.params.service)"
-              />
-            </SummaryView>
-          </RouterView>
-        </AppView>
-      </DataSource>
+                    <component
+                      :is="child.Component"
+                      :name="route.params.service"
+                      :service="data?.items.find((item) => item.name === route.params.service)"
+                    />
+                  </SummaryView>
+                </RouterView>
+              </DataCollection>
+            </template>
+          </DataLoader>
+        </KCard>
+      </AppView>
```

Comment:
This file didn't really need changing so mcu, but as I used this as a copy/paste starting point for the new listing page, I upgraded it to use newer patterns first, then did the copy/paste.